sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/parser/parsers.scala

```diff
@@ -99,6 +98,8 @@ abstract class AbstractParser extends DataTypeParserInterface with SQLConfHelper
           e.errorClass, e.messageParameters)
     }
   }
+
+  private def conf: SqlApiConf = SqlApiConf.get
```

Comment:
Maybe in the future it can be a trait `SqlApiConfHelper`

Reply:
> Maybe in the future it can be a trait `SqlApiConfHelper`

Should we do this in this pr?

Reply:
I am not going to add a trait if there is only one place where we are using it. For the record the original trait was added to make it easier to migrate existing code to the `SQLConf.get` based world. I don't really think adding it adds much value.